tests/test_comprehensive_idea_collection.py

```
@@ -0,0 +1,338 @@
+"""
+Tests for comprehensive idea collection from all generations.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)